configs/components/cfpropertylist.json

```diff
@@ -0,0 +1 @@
+{"url": "git@github.com:ckruse/CFPropertyList.git", "ref": "2.2.7"}
```

Comment:
had or hate?

Also, I'm not opposed but does the rewrite engine handle git sources?